src/helpers/clients.ts

```diff
@@ -1,8 +1,13 @@
 import { clients as Clients } from '@snapshot-labs/sx';
 
-const url: string = import.meta.env.VITE_MANA_URL || 'http://localhost:3000';
+const manaUrl: string =
+  import.meta.env.VITE_MANA_URL || 'http://localhost:3000';
+const ethUrl: string = import.meta.env.VITE_ETH_RPC_URL;
```

Comment:
Could you add this in the .env file:
```VITE_ETH_RPC_URL=https://rpc.ankr.com/eth_goerli```
Just to make it simpler for contributors to get setup